go-bridge-api/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+set -e
+
+ACCESS_TOKEN="${ACCESS_TOKEN:-access_token}"
+ACCOUNT_KEY="${ACCOUNT_KEY:-account_key}"
+CONVERSATION_KEY="${CONVERSATION_KEY:-conversation_key}"
+TRANSPORT_NAME="${TRANSPORT_NAME:-transport_name}"
+WEB_PATH="${WEB_PATH:-/go-bridge-api}"
+BASE_URL="${BASE_URL:-https://go.vumi.org/api/v1/go/http_api_nostream/}"
+REDIS_HOST="${REDIS_HOST:-127.0.0.1}"
+REDIS_DB="${REDIS_DB:-1}"
+REDIS_PORT="${REDIS_PORT:-6379}"
+AMQP_HOST="${AMQP_HOST:-127.0.0.1}"
```

Comment:
So you should be able to take out these `AMQP` things now, except for `AMQP_PREFETCH_COUNT`